BlazarUI/app/scripts/components/dashboard/StarredBranchesTableRow.jsx

```diff
@@ -9,6 +10,30 @@ import CommitMessage from '../shared/CommitMessage.jsx';
 
 class StarredBranchesTableRow extends Component {
 
+  constructor(props, context) {
+    super(props, context);
+  }
+
+  getRowClassNames(state) {
+    return classNames([
+      tableRowBuildState(state),
+      'clickable-table-row'
+    ]);
+  }
+
+  onTableClick(blazarBranchPath, blazarPath, e) {
+    if (e.target.className === 'sha-link') {
+      window.open(e.target.href, '_blank');
+    }
+
+    else if (e.target.className === 'repo-link') {
+      this.context.router.push(blazarBranchPath);
```

Comment:
now that we've upgraded react-router to v2 we should replace these with browserHistory

`browserHistory.push(blazarBrachPath)`